package/Dockerfile

```diff
@@ -26,22 +26,22 @@ RUN zypper --non-interactive install --no-recommends \
   which \
   libxml2-tools \
   libxslt-tools \
-  "rubygem(abstract_method)" \
-  "rubygem(cfa)" \
-  "rubygem(cheetah)" \
-  "rubygem(coveralls)" \
-  "rubygem(gettext)" \
-  "rubygem(parallel)" \
-  "rubygem(parallel_tests)" \
-  "rubygem(raspell)" \
-  "rubygem(rspec)" \
-  "rubygem(ruby:2.6.0:rubocop:0.41.2)" \
-  "rubygem(ruby:2.6.0:rubocop:0.71.0)" \
-  "rubygem(simplecov)" \
-  "rubygem(simpleidn)" \
-  "rubygem(suse-connect)" \
-  "rubygem(yard)" \
-  "rubygem(yast-rake)" \
+  "rubygem(ruby:2.7.0:abstract_method)" \
```

Comment:
cannot we use that system ruby macro? and call on it rpm eval? or in general at least get this version dynamic and use it here? e.g. by some ruby call?